[2022-12-06 13:22:45,934] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T13:21:55.483181+00:00 [queued]>
[2022-12-06 13:22:45,943] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T13:21:55.483181+00:00 [queued]>
[2022-12-06 13:22:45,943] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-12-06 13:22:45,943] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-12-06 13:22:45,943] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-12-06 13:22:45,955] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): LoadPositionsInMongoDB> on 2022-12-06T13:21:55.483181+00:00
[2022-12-06 13:22:45,958] {standard_task_runner.py:52} INFO - Started process 368 to run task
[2022-12-06 13:22:45,962] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_mongo_db_dag', 'LoadPositionsInMongoDB', '2022-12-06T13:21:55.483181+00:00', '--job-id', '16', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_mongo_db_dag.py', '--cfg-path', '/tmp/tmpqumtych_', '--error-file', '/tmp/tmpnjb30i_k']
[2022-12-06 13:22:45,962] {standard_task_runner.py:77} INFO - Job 16: Subtask LoadPositionsInMongoDB
[2022-12-06 13:22:46,002] {logging_mixin.py:104} INFO - Running <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T13:21:55.483181+00:00 [running]> on host 2e62a70a5623
[2022-12-06 13:22:46,046] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_mongo_db_dag
AIRFLOW_CTX_TASK_ID=LoadPositionsInMongoDB
AIRFLOW_CTX_EXECUTION_DATE=2022-12-06T13:21:55.483181+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-06T13:21:55.483181+00:00
[2022-12-06 13:22:46,170] {logging_mixin.py:104} INFO - debut de chargement des données dans mongodb
[2022-12-06 13:22:46,346] {logging_mixin.py:104} INFO - fin de chargement des données dans mongodb
[2022-12-06 13:22:46,349] {python.py:151} INFO - Done. Returned value was: None
[2022-12-06 13:22:46,359] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_mongo_db_dag, task_id=LoadPositionsInMongoDB, execution_date=20221206T132155, start_date=20221206T132245, end_date=20221206T132246
[2022-12-06 13:22:46,380] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-06 13:22:46,415] {local_task_job.py:151} INFO - Task exited with return code 0
